packages/@aws-cdk/aws-ecs/test/test.ecs-cluster.ts

```diff
@@ -636,6 +636,59 @@ export = {
     test.done();
   },
 
+  "configures userdata with powershell if windows machine image is specified"(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.Vpc(stack, 'MyVpc', {});
+
+    const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+    cluster.addCapacity('WindowsAutoScalingGroup', {
+      instanceType: new ec2.InstanceType('t2.micro'),
+      machineImage: new ecs.EcsOptimizedAmi({
+        windowsVersion: ecs.WindowsOptimizedVersion.SERVER_2019,
+      }),
+    });
+
+    // THEN
+    expect(stack).to(haveResource("AWS::AutoScaling::LaunchConfiguration", {
+      ImageId: {
+        Ref: "SsmParameterValueawsserviceecsoptimizedamiwindowsserver2019englishfullrecommendedimageidC96584B6F00A464EAD1953AFF4B05118Parameter"
+      },
+      InstanceType: "t2.micro",
+      IamInstanceProfile: {
+        Ref: "EcsClusterWindowsAutoScalingGroupInstanceProfile65DFA6BB"
+      },
+      SecurityGroups: [
+        {
+          "Fn::GetAtt": [
+            "EcsClusterWindowsAutoScalingGroupInstanceSecurityGroupDA468DF1",
+            "GroupId"
+          ]
+        }
+      ],
+      UserData: {
+        "Fn::Base64": {
+          "Fn::Join": [
+            "",
+            [
+              "<powershell>Remove-Item -Recurse C:\\ProgramData\\Amazon\\ECS\\Cache\nImport-Module ECSTools\n[Environment]::SetEnvironmentVariable(\"ECS_CLUSTER\", \"",
```

Comment:
For my own edification, where are the `<powershell>` tags being specified? I'm not seeing it in the changes to `cluster.ts`.